nixos/tests/hydra/common.nix

```diff
@@ -36,13 +36,6 @@
       '';
     };
     services.postfix.enable = true;
-    nix = {
-      distributedBuilds = true;
-      buildMachines = [{
-        hostName = "localhost";
-        systems = [ system ];
```

Comment:
It feels weird to me that this 'fixes' the tests. Is something broken in these settings upstream? I'm a bit confused. But great that at least a working configuration is found.

Reply:
I think remote machines are somehow differently configured/merged and localhost is no longer a valid remote builder or so. I am not sure, the symptom was, that hydra couldn't build things and they failed with no error message.